### DISCUSSION TITLE
Errors found on the switch from Terria v7 to v8

### DISCUSSION THREAD

**Discussion by LorenzoStucchi** [TerriaJS/terriajs#5777] [Show and tell] [answered]
I found some issues with the switch of the version, I open some issues here:

Feature information tool does not keep the cql_filer setting:
https://github.com/TerriaJS/TerriaMap/issues/552

Increase the height in the branding section:
https://github.com/TerriaJS/TerriaMap/issues/551

**Comment by tephenavies** [comment, marked as answer]
Hi @LorenzoStucchi. Thanks for writing those up
